testsuite/features/step_definitions/common_steps.rb

```diff
@@ -5,11 +5,6 @@
 require 'securerandom'
 require 'pathname'
 
-When(/^I save a screenshot as "([^"]+)"$/) do |filename|
-  save_screenshot(filename)
-  attach File.open(filename, 'rb'), 'image/png'
-end
-
```

Comment:
Let's keep that one please!
I use that from time to time while debugging scenarios.
You can add a comment on top of it about using this only for debugging or so 👼🏼